CONTRIBUTING.md

```diff
@@ -30,6 +30,7 @@ file.
     - Change the "Internal Java Platform" to `IntelliJ IDEA jbr 11`
     - Extend it with additional plugin libraries by adding to `Classpath`:
       - plugins/git4idea/lib/git4idea.jar
+      - plugins/android/lib/android.jar
```

Comment:
I had to add this because of the following error - 
```
/mnt/common-data/study-material/flutter/flutter-intellij/src/io/flutter/ProjectOpenActivity.java:34:35
java: package org.jetbrains.android.facet does not exist
/mnt/common-data/study-material/flutter/flutter-intellij/src/io/flutter/ProjectOpenActivity.java:107:40
java: cannot find symbol
  symbol:   class AndroidFrameworkDetector
  location: class io.flutter.ProjectOpenActivity
```

Reply:
Thanks, I forgot to update that.